### PR TITLE
Add `--runs-per-job`, `--max-run-fails-per-job`, and more to `metadata` cmd

### PR DESCRIPTION
**Pull request by wslulciuc**
This PR adds the following cmd line args to [`cli.MetadataCommand`](https://github.com/MarquezProject/marquez/blob/main/api/src/main/java/marquez/cli/MetadataCommand.java) used to seed the Marquez backend for future functional testing:

Screenshot 2024-10-14 at 6 14 03 PM

### CLI Args

**`--jobs`**

```
limits OL jobs up to N (default: 5) -- replaces `--runs`
```

**`--runs-per-job`**

```
 limits OL run executions per job up to N (default: 10)
```

**`--runs-active`**

```
limits OL run executions marked as active (='RUNNING') up to N
```

**`--max-run-fails-per-job`**

```
 maximum OL run fails per job (default: 2)
```

**`--min-run-duration`**

```
minimum OL run duration (in seconds) per execution (default: 300)
```

**`--max-run-duration`**

```
maximum OL run duration (in seconds) per execution (default: 900)
```

**`--run-start-time`**

```
 specifies the OL run start time in UTC ISO ('YYYY-MM-DDTHH:MM:SSZ');
 used for the initial OL run, with subsequent runs starting relative to the
initial start time. (default: 2024-10-15T01:00:11.080828Z)
```

**`--run-end-time`**

```
specifies the OL run end time in UTC ISO ('YYYY-MM-DDTHH:MM:SSZ');
used for the initial OL run, with subsequent runs ending relative to the
initial end time. (default: 2024-10-15T01:07:25.080828Z)
```

### Example

```
java -jar marquez.jar metadata \
  --jobs 10 \
  --runs-per-job 5 \
  --max-run-fails-per-job 2
Generating runs '5' per job, each COMPLETE/FAIL run event will have a size of '~33404' (bytes)...
Writing '100' events to: 'metadata.json'
```
_output_: [metadata.json](https://github.com/user-attachments/files/17370791/metadata.json)

## Bugs

This PR also fixes the following bugs:

* Display last `N` runs for job in dashboard page; replaces randomized run durations
* Orders `N` runs for job in dashboard page as ascending
* Fix latest run state color to use `runStateColor()` for job display page

### CODE REVIEW

File: api/src/main/java/marquez/cli/MetadataCommand.java [hunk 1]
```diff
@@ -343,8 +410,8 @@ private static ZonedDateTime newNominalTime() {
   }
 
   /** Returns a new {@code event} time. */
-  private static ZonedDateTime newEventTime() {
-    return Instant.now().atZone(AMERICA_LOS_ANGELES);
+  private static ZonedDateTime newEventTimeAsUtc() {
+    return ZonedDateTime.now(UTC);
```
Comment (phixMe) [inline api/src/main/java/marquez/cli/MetadataCommand.java:414]: Nice!

**Review by phixMe — APPROVED** [review]
Looks good!

**Comment by phixMe** [comment]
Is there any way to get a job in running status so that we could look at those states. Maybe a flag could be added as, `current-job-running`?

**Comment by phixMe** [comment]
It also looks like you need to clean up a CI error on the web project.

**Comment by wslulciuc** [comment]
> Is there any way to get a job in running status so that we could look at those states. Maybe a flag could be added as, current-job-running?

Do you mean to print out a list of jobs / runIDs to the terminal after running the cmd?

**Comment by phixMe** [comment]
> > Is there any way to get a job in running status so that we could look at those states. Maybe a flag could be added as, current-job-running?
> 
> ~Do you mean to print out a list of jobs / runIDs to the terminal after running the cmd?~ Hmm you mean to have job runs in a `RUNNING` state?

Yes, to let the last run for remain in running state...

**Review by phixMe — APPROVED** [review]

**Pull request merged**